tests/test_routes.py

```diff
@@ -188,17 +189,12 @@ def test_delete_inventory_record_success(self):
         # check if record was created successfully
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
-        request_body = {
-            "product_id": test_inventory_record.product_id,
-            "condition": test_inventory_record.condition.value
-        }
-
-        response = self.client.delete(f"{BASE_URL}/{test_inventory_record.product_id}", json=request_body)
+        response = self.client.delete(f"{BASE_URL}/{test_inventory_record.product_id}")
```

Comment:
I think we should pass condition as well in the url

Reply:
replied to previous comment!